qa/standalone/erasure-code/test-erasure-eio.sh

```diff
@@ -154,8 +154,10 @@ function inject_eio() {
     local -a initial_osds=($(get_osds $poolname $objname))
     local osd_id=${initial_osds[$shard_id]}
     set_config osd $osd_id filestore_debug_inject_read_err true || return 1
-    CEPH_ARGS='' ceph --admin-daemon $(get_asok_path osd.$osd_id) \
-             injectdataerr $poolname $objname $shard_id || return 1
+    while ( CEPH_ARGS='' ceph --admin-daemon $(get_asok_path osd.$osd_id) \
+             injectdataerr $poolname $objname $shard_id | grep -q Invalid ); do
+        sleep 1
```

Comment:
@dzafman 
Might want to add something of a timeout counter?
Especially if things are executed standalone, it will take forever to run all these scripts.
And so I push and forget, but failing here would just burn time.